spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/features/FeaturesProperties.java

```diff
@@ -33,7 +33,7 @@
 
     public static final String ANALYTICS_ENABLED = "analytics-enabled";
 
-    private boolean analyticsEnabled = true;
+    private boolean analyticsEnabled;
```

Comment:
maybe explicitly setting to `false` would be more clear (from a self-documenting perspective)